src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsChannelMappingTest.java

```diff
@@ -119,7 +121,7 @@ private static Stream<Arguments> hearingChannelArguments() {
             Arguments.of(YES.getValue(), YES.getValue(), YES.getValue(), FACE_TO_FACE),
             Arguments.of(NO.getValue(), YES.getValue(), NO.getValue(), HearingChannel.VIDEO),
             Arguments.of(NO.getValue(), YES.getValue(), YES.getValue(), HearingChannel.VIDEO),
```

Comment:
Please use static import on all these HearingChannel values and check whole Class for obsolete qualifiers

Reply:
done